tests/integ/modin/index/test_index_methods.py

```diff
@@ -400,3 +400,22 @@ def test_create_index_from_df_negative():
         pd.Index(pd.DataFrame([[1, 2], [3, 4]]))
     with pytest.raises(ValueError):
         pd.DatetimeIndex(pd.DataFrame([[1, 2], [3, 4]]))
+
+
+@sql_count_checker(query_count=4, join_count=4)
+def test_identical():
+    i1 = pd.Index(["a", "b", "c"])
+    i2 = pd.Index(["a", "b", "c"])
+
+    assert i1.identical(i2)
```

Comment:
why not compare to native pandas?

Reply:
this is the test copied from pandas too.